wgsl/index.bs

```diff
@@ -8411,6 +8418,32 @@ path: syntax/binding_attr.syntax.bs.include
 
 </table>
 
+## `blend_src` ## {#blend-src-attr}
+
+<pre class=include>
+path: syntax/blend_src_attr.syntax.bs.include
+</pre>
+
+<table class='data builtin'>
+  <caption><dfn noexport dfn-for="attribute">`blend_src`</dfn> Attribute</caption>
+  <tr>
+    <td style="width:10%">Description
+    <td>
+    Specifies a part of the [=fragment=] output when the feature [=extension/dual_source_blending=]
+    is enabled.
+    See [[#input-output-locations]].
+
+    [=shader-creation error|Must=] only be applied to a member of a [=structure=] type.
```

Comment:
Should this also say

> That member must also have a `@location` attribute.

This would restrict the ability to add `@blend_src` in arbitrary but useless places.

Reply:
Done

Reply:
Hi @dneto0, could you take a look?